src/components/application_manager/src/commands/mobile/add_command_request.cc

```diff
@@ -187,10 +187,12 @@ void AddCommandRequest::Run() {
   }
 
   if (send_ui_) {
+    StartAwaitForInterface(HmiInterfaces::HMI_INTERFACE_UI);
```

Comment:
@okozlovlux can it be moved to SendHMIRequest? It is better to make mapping of interfaces andmethod of this interfaces, that inserting additional string to EACH command